src/App.tsx

```diff
@@ -14,7 +17,9 @@ function App() {
         <h1 className="text-center">Centered?</h1>
     </div>
       <div>
+        <Navbar></Navbar>
         <Hero></Hero>
+        <News></News>
        <SponsorSection></SponsorSection>
```

Comment:
Om inte dessa komponenter ska ha några children, dvs: `<Navbar>children</Navbar>`
Så kan man göra som nedan

```suggestion
   <Navbar />
   <Hero />
   <News />
   <SponsorSection />
```